GUI/Main.cs

```diff
@@ -285,6 +285,8 @@ protected override void OnLoad(EventArgs e)
         {
             Location = configuration.WindowLoc;
             Size = configuration.WindowSize;
+            splitContainer1.SplitterDistance = configuration.PanelPosition;
```

Comment:
Our method to save/restore the window size doesn't seem two work for maximized windows on Windows (I wanted to play some games over the winter). If I maximize and then move the splitter to be a reasonable distance from the right edge, closing and relaunching gives me a squished right pane in a smaller window, presumably because it's trying to set the pane width to greater than the actual window:

![image](https://user-images.githubusercontent.com/1559108/34360257-d5037384-ea56-11e7-9b39-0d6c8c51245d.png)

Reply:
If you then maximise, is the pane width correct?

Reply:
Ah, got it. It's only storing the Window Size when the Window is not maximized, but not storing the fact that the window is maximized. Linux doesn't actually maximize the window, it just makes it fir the window space, so we don't see the issue there.